exspy/exspy/components/eels_arctan.py

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8 -*-
-# Copyright 2007-2023 The HyperSpy developers
+# Copyright 2007-2023 The exspy developers
```

Comment:
In some file headers, `exSpy` is capitalized correctly, but in most it is still all lower case.